src/error_styles-nomodules.css

```diff
@@ -0,0 +1,8 @@
+.globalErrorContainer {
+    width: 40%;
+    margin-top: 10%;
+}
+
+.globalErrorContainer a.btn {
```

Comment:
If you remove the '-nomodules.css' extension, and if `a.btn` is to reference a bootstrap/patternfly button, then you would need to rewrite the css as:
````css
.globalErrorContainer a:global(.btn) {
 ...
}
````

`:global()` is a directive to the CSS module loader to leave that classname untouched.

Reply:
thanks